### ISSUE TITLE
Use the entire advanced search form for querying items

### ISSUE THREAD

**Issue by jeremyboggs** [scholarslab/Neatline#262]
Neatline currently only provides a very limited set of fields for importing Omeka items, and leaves off quite a few fields that would be useful, like importing items added by a particular user. I haven't looked into it, but I'm guessing this also means any additional fields added by plugins are also skipped.

Unless there's some technical reason not to do this, I'd recommend just using the default item search form in Omeka. This has the advantage of replicating a search form users probably have already encountered, and would also pull in any fields added by other plugins.

**Comment by tamielbr** [comment]
+1

**Comment by davidmcclure** [comment]
Yep, this would actually be easier and more maintainable than what we have now (a custom form that duplicates parts of the advanced search form). Looks like there's even a [helper function](https://github.com/omeka/Omeka/blob/master/application/libraries/globals.php#L2216) that does it all automatically.
